### PR TITLE
Remove font-family

### PR DESCRIPTION
**Pull request by nskrypnik**
# PATCH

### CODE REVIEW

**Comment by sandersky** [comment]
:thumbsup:

File: tests/dummy/app/styles/app.scss [hunk 1]
```diff
@@ -18,7 +17,6 @@
     white-space: -o-pre-wrap;
   }
   code {
-    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
```
Comment (job13er) [inline tests/dummy/app/styles/app.scss:21]: hmm, this is apparently for code samples in the demo
do we really want to remove this?

Comment (sandersky) [inline tests/dummy/app/styles/app.scss:21]: Shouldn't demos follow our desired UX which includes the correct font?

Comment (job13er) [inline tests/dummy/app/styles/app.scss:21]: I was thinking we were removing the `font-family` settings in the components themselves, not the demos. Then again, we should probably have `ember-frost-css-theme` or whatever provide the right styles for code samples... so I guess this is fine.

Comment (job13er) [inline tests/dummy/app/styles/app.scss:21]: I don't think there is UX defined for code samples, but they better be monospace fonts if you don't wanna drive people nuts

**Pull request closed without merging**